backend/MeetApp/src/main/java/com/meetapp/meetapp/configuration/SampleDataLoader.java

```diff
@@ -138,7 +138,7 @@ private List<Meeting> getMeetings() {
                         "Nocny przejazd przez centrum Wrocławia w styczniu.",
                         "Kochani zapraszam Was na epicki przejazd centrum Wrocławia w Sobotę 7 stycznia!!! 💪💪🚩💯 " +
                                 "Zaczynamy o 18:00. Czołówki obowiązkowe ;) Zbiórka przed NFM.",
-                        Instant.parse("2023-01-07T18:00:00.000Z"), getCategories(new HashSet<>(Arrays.asList(3))), 35),
+                        Instant.parse("2022-11-24T20:50:00.000Z"), getCategories(new HashSet<>(Arrays.asList(3))), 35),
```

Comment:
I guess we can keep it so that there is always an expired post in the db. But you could change the dates in the description :laughing: